apps/estuary/config/test.exs

```diff
@@ -1 +1,5 @@
 use Mix.Config
+
+config :estuary,
+  elsa_endpoint: [localhost: 9092],
+  event_stream_topic: "event-stream"
```

Comment:
I dont think we need this anymore

Reply:
We will need this if we move the dev.exs to integration.exs, but not if we move it to config.exs

Reply:
Will the dev.exs config only get used if we set `MIX_ENV=dev`? Do we currently do that for any environments? I've only seen the `MIX_ENV` set to integration

Reply:
dev.exs is the default environment for iex -S mix.  If these properties are in config, they're used in all environments.  If they're in integration, they're only used in integration.  Typically we only put these in integration and actually i mis spoke you dont have unit tests that need these so they dont need to be in test.exs either way.

Reply:
Per my comment above, you probably want to keep these values out of the root config file, but that being said, if you focus your tests accordingly (and possibly write conditionals to _not_ start certain components like elsa outside of integration or prod) you can just omit these values from the config file for that environment.